.github/actions/download-built-package/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Download built package from cache
+description: Download built package from cache
+runs:
+  using: "composite"
+  steps:
+  - name: Download built package from cache
```

Comment:
re-use the built assets from the first C.I. job so we reduce variability between jobs, and ensure we work with the same assets throughout the C.I. run -- this is as close to simulating a package release (without releasing), as I could come up with